lang/c/c_pointers.talon-list

```diff
@@ -0,0 +1,6 @@
+list: user.c_pointers
+code.language: c
+code.language: cpp
```

Comment:
Please double check, but I do not think that C++ actually makes use of this list. C++ has its own list that includes references and the likes.